src/lib/y2network/widgets/interface_type.rb

```diff
@@ -0,0 +1,41 @@
+require "yast"
+require "cwm/common_widgets"
+
+Yast.import "NetworkInterfaces"
+
+module Y2Network
+  module Widgets
+    class InterfaceType < CWM::RadioButtons
+      attr_reader :result
+      def initialize
+        textdomain "network"
+      end
+
+      def label
+        _("&Device Type")
+      end
+
+      def help
+        # FIXME: help is not helpful
+        _(
+          "<p><b>Device Type</b>. Various device types are available, select \n" \
+            "one according your needs.</p>"
```

Comment:
```suggestion
            "one according to your needs.</p>"
```

Reply:
This is basically just copy to not break translation. As you can see in fixme, we really really should write there something useful. E.g. about types of cards.